manifests/candlepin.pp

```diff
@@ -49,6 +70,13 @@
 
   if $deploy {
 
+    Cert[$tomcat_cert_name] ~>
```

Comment:
For the above, I'm fine with filing Github issues to track these unless you feel we should push to Redmine to collect them all under a single tracker.